code/game/objects/structures/signs.dm

```diff
@@ -815,4 +815,10 @@
 /obj/structure/sign/poi/engineright
 	name = "I.C.V."
 	desc = "The charred name of a cargo ship of some description."
-	icon_state = "poi_engine2"
\ No newline at end of file
+	icon_state = "poi_engine2"
+
+/obj/structure/sign/coffee

```

Comment:
//yw edit begin and end pls

Reply:
Just found out that we actually have a seperate file for new signs, that was apparently disabled.
Gonna move this there.

Reply:
Sounds good to me